core/src/test/java/org/mskcc/cbio/portal/scripts/TestIntegrationTest.java

```diff
@@ -130,7 +130,7 @@ public void testLoadStudyEs0() throws Throwable {
                 }
             }
             //check that there are only warnings for empty positions in fake data:
-            assertEquals(13, countWarnings);
+            assertEquals(12, countWarnings);
```

Comment:
Let's revert this to an assert that checks that there are no warnings. To get this working, you will also need to remove the cause of the warnings from study_es_0. I see now that the confusion was introduced by @dionnezaal in commit https://github.com/cBioPortal/cbioportal/commit/bbbdae3bef9c8dd95e9e2b34e51abe98a90cd834 . The solution for these extra 12 warnings will be to fix the study_es_0 MAF data so that it does not give warnings. 

Original code was: 
```java
          //check that there no warnings:
          assertEquals(0, countWarnings);
```

Reply:
Apologies for the late reply. I fixed this in a PR to @n1zea144 's branch: https://github.com/n1zea144/cbioportal/pull/1

I also reverted many floats back to integers, for example for Start_Position, which was accidentally done in cBioPortal/cbioportal@bbbdae3#diff-3cd4ce7cd10421c2d456537975d4d810 , when adding the Germline mutations.